app/src/main/java/by/paulk/mycalc/calculator/Calculator.kt

```diff
@@ -8,18 +8,17 @@ class Calculator : ICalculator {
     override  fun calculate(input: String):String{
 
         val expression = ExpressionBuilder(input).build()
-        try {
+        return try {
             val result = expression.evaluate()
 
             if ((result % 1) == 0.0){
-                return result.toInt().toString()
-            }
-            else{
-                return result.toString()
+                result.toInt().toString()
+            } else{
+                result.toString()
             }
         } catch (ex: ArithmeticException) {
 
-            return "ERROR"
+            "ERROR"
```

Comment:
all strings digits (except 0, 1/-1) should be extracted to constant
Each string also should be checked if translation is needed in most cases it should be and in that cases move it into resources